### PR TITLE
Solve holding tier issue for instances

### PR DESCRIPTION
**Pull request by nesitor**
Fix: We are not managing well the holding tier on instances, so will be better to disable that control.

### CODE REVIEW

**Review by MHHukiewitz — APPROVED** [review]
LGTM, it triggered the stopping of the scheduled holder-tier VM, so we disabled it temporarily

File: src/aleph/vm/orchestrator/tasks.py [hunk 1]
```diff
@@ -147,20 +147,20 @@ async def monitor_payments(app: web.Application):
...
+        #
+        #         # Stop executions until the required balance is reached
+        #         required_balance = await compute_required_balance(executions)
+        #         logger.debug(f"Required balance for Sender {sender} executions: {required_balance}")
+        #         # Stop executions until the required balance is reached
+        #         while balance < (required_balance + settings.PAYMENT_BUFFER):
+        #             last_execution = executions.pop(-1)
+        #             logger.debug(f"Stopping {last_execution} due to insufficient balance")
+        #             await pool.stop_vm(last_execution.vm_hash)
+        #             required_balance = await compute_required_balance(executions)
```
Comment (hoh) [inline src/aleph/vm/orchestrator/tasks.py:163]: Why not remove the code completely ?

Comment (MHHukiewitz) [reply src/aleph/vm/orchestrator/tasks.py:163]: We might write some test in the future that allows us to correctly test this and re-enable it as a safety feature

**Comment by MHHukiewitz** [comment]
Btw @hoh can you also check the LLAMA_URL secret in this repository? It seems to work on [aleph-sdk-python](https://github.com/aleph-im/aleph-sdk-python/pulls) with the Libertai API

**Pull request merged**